### PR TITLE
Ret 2195

### PR DESCRIPTION
**Pull request by olyode**
### JIRA link
https://tools.hmcts.net/jira/browse/RET-2195
### Change description
Final checks
Added label to average weekly hours input field
updated unit test
Updated average weekly hours hint text
**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

File: src/test/unit/views/average-weekly-hours.test.ts [hunk 1]
```diff
@@ -40,6 +41,12 @@ describe('Average weekly hours page Still Working', () => {
     expect(title[0].innerHTML).contains(expectedTitle, 'Page title does not exist');
   });
 
+  it('should display input field label', () => {
+    const expectedInputLabel = averageWeeklyHoursJson.enterAverageHours;
+    const label = htmlRes.getElementsByClassName(labelClass);
+    expect(label[0].innerHTML).contains(expectedInputLabel, 'Average weekly hour inut label does not exist');
```
Comment (Itay-Kirsh) [inline src/test/unit/views/average-weekly-hours.test.ts:47]: "...hour in**p**ut label..."

**Review by DarylBoam — DISMISSED** [review]

File: src/main/resources/locales/en/translation/notice-length-months.json [hunk 2]
```diff
@@ -9,7 +9,7 @@
   "errors": {
     "noticePeriodLength": {
       "invalid": "Tell us the number of months in your notice period",
-      "notANumber": "Number of months in notice period must be a number, like 30"
```
Comment (Itay-Kirsh) [inline src/main/resources/locales/en/translation/notice-length-months.json:12]: etc with a full stop afterwards

File: src/main/resources/locales/en/translation/notice-length-weeks.json [hunk 3]
```diff
@@ -9,7 +9,7 @@
   "errors": {
     "noticePeriodLength": {
       "invalid": "Tell us the number of weeks in your notice period",
-      "notANumber": "Number of weeks in notice period must be a number, like 30"
```
Comment (Itay-Kirsh) [inline src/main/resources/locales/en/translation/notice-length-weeks.json:12]: etc with a full stop afterwards

**Review by Itay-Kirsh — CHANGES_REQUESTED** [review]

**Comment by olyode** [comment]
PR issues, rebased to another branch

**Pull request closed without merging**